library/lib.ml

```diff
@@ -411,8 +411,8 @@ type abstr_info = {
 type abstr_list = abstr_info Names.Cmap.t * abstr_info Names.Mindmap.t
 
 type secentry =
-  | Variable of (Names.Id.t * Decl_kinds.binding_kind *
-		   Decl_kinds.polymorphic * Univ.ContextSet.t)
+  | Variable of (Names.Id.t * Decl_kinds.binding_kind * bool * Univ.ContextSet.t)
+  (** (name, kind, poly, univs) *)
```

Comment:
Record?

Reply:
Sure, feel free to push or to suggest a naming scheme.